### ISSUE TITLE
some packages will download a bunch of System.* reference

### ISSUE THREAD

**Issue by kaynn** [GlitchEnzo/NuGetForUnity#320]
add Protobuf-net assembly to nuget.
the NugetForUnity dll will download 92 System.* packages, which clashes with those from unity.

- **NuGet Package:** Protobuf-net
- **NuGetForUnity Version:** 2.0
- **Unity Version:** 2019.2.15f1
- **Operating System:** windows 10 pro

**Comment by oliverbooth** [comment]
+1 for this, also by installing protobuf-net. I attempted to remove all the System.* packages but NuGetForUnity seems to just redownload them all again.

**Comment by Efmi** [comment]
This is the actual error message. Seems like there is only a conflict with two packages: ```System.Xml.XmlSerializer.dll``` & ```System.Threading.dll```
![image](https://user-images.githubusercontent.com/22080933/71634117-ea9e3080-2c21-11ea-8c0f-74a07eba16ac.png)

**Comment by oliverbooth** [comment]
I found a possible workaround until this issue is fixed:

Delete all the conflicting System libraries that get imported. Don't delete the folders, or the nupkg file or any other metadata files. Just delete the conflicting .dll files (**and only the .dll files**) inside each lib/ directory. This'll stop NuGetForUnity redownloading everything.

Obviously this isn't an ideal solution but for now it's the best we got

**Comment by DamodarSojka** [comment]
Still an issue, when installing mongodb driver over 70 packages were imported.

**Comment by neilyio** [comment]
This sounds related to #328, I chimed in there because I had issues with System.* packages trying to install language-ext.

The solution there worked for me: changing the API compatibility to .NET 4.x instead of .NET standard 2.0 in Project Settings > Player (I had to do this before adding any files to a brand new Unity project to get it to work).

It's really not ideal, however, because I'm finding that .NET 4.x makes compilation around 10x slower. I'd love to see a solution, I've just decided to forego installing external packages in the meantime.
